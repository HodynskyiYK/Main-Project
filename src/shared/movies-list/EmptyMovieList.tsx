import React, { FC } from 'react'
import styles from './EmptyMovieList.module.scss'

export const EmptyMovieList: FC = () => {
    return (
        <div className={styles.emptyList}>
            <p className="h1">WE WERE SEARCHING, SEARCHING, BUT NOT FOUND.</p>
            <p>Try to change the request, or choose something interesting from our selections</p>
        </div>
    )
}
