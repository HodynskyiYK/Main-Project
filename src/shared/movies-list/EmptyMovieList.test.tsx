import React from 'react'
import { render } from '@testing-library/react'
import { EmptyMovieList } from './EmptyMovieList'

describe('EmptyMovieList tests', () => {

    test('To match snapshot', () => {
        const {asFragment} = render(<EmptyMovieList/>)
        expect(asFragment()).toMatchSnapshot()
    })
})