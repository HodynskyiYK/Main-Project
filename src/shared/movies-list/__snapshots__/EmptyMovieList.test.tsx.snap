// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EmptyMovieList tests To match snapshot 1`] = `
<DocumentFragment>
  <div
    class="emptyList"
  >
    <p
      class="h1"
    >
      WE WERE SEARCHING, SEARCHING, BUT NOT FOUND.
    </p>
    <p>
      Try to change the request, or choose something interesting from our selections
    </p>
  </div>
</DocumentFragment>
`;
